conf/CFBG.conf.dist

```diff
@@ -24,6 +24,11 @@
 #       Description: Enable check average player level for bg
 #       Default: 1
 #
+#   CFBG.BalancedTeams.LowLevelClass
+#       Description: Enable class balance for BGs 10-19
+#       Default: 0
+#                1 - (Enabled. It puts Hunters in the weaker team)
```

Comment:
not sure if it's badly described or just badly implemented, but it should not be about "putting hunters in the weaker team", rather it should be "split hunters equally in the teams". So a team cannot have +2 hunters compared to the other one.